Classes/Hooks/TypolinkHook.php

```diff
@@ -23,11 +24,25 @@
 class TypolinkHook
 {
     /**
+     * This Hook will convert typolinks to be used in Vue/Nuxt frontend,
+     * by modifying `lastTypoLinkUrl` with json encoded details about the previously generated typolink.
+     * If the static TypoScript Template won't be used, the default typolink behavior will be retained.
+     *
      * @param array $params
      * @param ContentObjectRenderer $ref
      */
     public function handleLink(array $params, ContentObjectRenderer $ref): void
     {
+
+        /** @var array $setup */
```

Comment:
I think we could get rid of this, because it's declared in TemplateService already, so IDE knows what kind of var it is